soccer/gameplay/visualization/robot_viz.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Optional, Tuple
+import robocup
+from math import sin, cos
+
+
+def draw_heading(robot: robocup.Robot,
+                 debug_drawer: robocup.DebugDrawer,
+                 r: float = 0.5,
+                 color: Tuple[int, int, int] = (0xff, 0xc1, 0xda)):
+    """ Draws a line representing the heading of the robot
+    :param robot: Robots to draw heading for
+    :param debug_drawer: DebugDrawer
+    :param r: How long to draw the heading line
+    :param color: What color to draw the heading line
+    """
+    theta = robot.angle
+    endpoint = robot.pos + robocup.Point(r * cos(theta), r * sin(theta))
```

Comment:
You should be able to do r*robocup.Point

Reply:
There's a robocup.Point.Direction(theta) that does the cos and sin automatically